cmd/install/install.go

```diff
@@ -229,3 +241,25 @@ func downloadArgoCD(osPlatform string) error {
 	}
 	return nil
 }
+
+func installCAPI(ver string) error {
+	c, err := client.New("")
+	if err != nil {
+		return err
+	}
+	options := client.InitOptions{
+		Kubeconfig:              client.Kubeconfig{Path: kubeconfigPath},
+		CoreProvider:            ver,
+		BootstrapProviders:      nil,
+		InfrastructureProviders: nil,
+		ControlPlaneProviders:   nil,
+		TargetNamespace:         "",
+		LogUsageInstructions:    true,
+		WaitProviders:           false,
+		WaitProviderTimeout:     time.Second * 5 * 60, // this is the default for clusterctl
+	}
+	if _, err := c.Init(options); err != nil {
```

Comment:
What happens if CAPI is already installed? Is this idempotent?

Reply:
Yeah. Just like clusterctl init

Reply:
However, it'll show this message
```shell
Attempting to install cluster-api:v1.1.5 with infrastructure providers [] and bootstrap providers []
Error: installing provider "cluster-api" can lead to a non functioning management cluster: there is already an instance of the "cluster-api" provider installed in the "capi-system" namespace
```